demos/connector/neo4j/undirected-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -47,8 +47,10 @@
    "source": [
```

Comment:
<p>not sure where this "['...']" output comes from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1046/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>https://github.com/stellargraph/stellargraph/issues/826</p>